### PR TITLE
Wiki edit

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: shuttleofx_catalog/views.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ def deleteBundle(bundleId):
     if bundle == None:
         abort(make_response("Bundle not found.", 404))
 
-    plugins = config.pluginTable.find({"bundleId": bundleId})
```
Comment (Olivier-Daire) [inline shuttleofx_catalog/views.py:159]: Why ?

File: shuttleofx_client/static_tmp/styles/wiki.less [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#metaWiki fieldset {
```
Comment (Olivier-Daire) [inline shuttleofx_client/static_tmp/styles/wiki.less:1]: Remove unused code lines

**Comment by Olivier-Daire** [comment]
I have some comments about the editor : 
-  when entering code inside ``` the indentation is not kept, and the syntax highlighting doesn't seems to work
- already said it but the preview button is not very well placed, in my mind, plus I think that there should be a change when you are in preview vs when you are in edit mode
- why is there a `<h3>Preview :</h3>` below the editor ? [EDIT] ok got it, it's when you save, but it think it kind of weird when you edit it
- a small padding on the left between the text and the input border would be great
- fullscreen mode is very cool ! But the preview button text goes black when we go into preview mode
- would be great if we could use `CTRL Z` but this may be complicated

File: shuttleofx_catalog/views.py (hunk 1 above)
Comment (ghost) [inline shuttleofx_catalog/views.py:159]: I don't know !! this is a terrible error omg !

File: shuttleofx_client/static_tmp/styles/wiki.less (hunk 2 above)
Comment (ghost) [inline shuttleofx_client/static_tmp/styles/wiki.less:1]: done :+1:

**Pull request closed without merging**